baseplate/observers/prom_metrics_tagged.py

```diff
@@ -0,0 +1,197 @@
+from random import random
+from time import time
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from prometheus_client import Counter
+from prometheus_client import Histogram
+
+from baseplate import _ExcInfo
+from baseplate import BaseplateObserver
+from baseplate import LocalSpan
+from baseplate import RequestContext
+from baseplate import Span
+from baseplate import SpanObserver
+from baseplate.lib import config
+
+
+class _PrometheusMetrics:
+    def __init__(self, allowlist: List[str]):
+        self.server_rate = Counter(
+            "baseplate_server_span_rate", "Baseplate Server Span Counter", allowlist
```

Comment:
These were temporarily taken out but would end up matching this commented block
https://github.snooguts.net/reddit/baseplate.spec/blob/master/component-apis/prom-metrics.md?plain=1#L442